src/openrct2/ride/Vehicle.cpp

```diff
@@ -7608,25 +7610,22 @@ bool Vehicle::UpdateTrackMotionForwards(CarEntry* carEntry, Ride* curRide, rct_r
             }
         }
     }
-    else if (trackType == TrackElemType::Booster)
+    else if (
+        (trackType == TrackElemType::Booster)
+        || (trackType == TrackElemType::Flat && curRide->type == RIDE_TYPE_REVERSE_FREEFALL_COASTER)
+        || (trackType == TrackElemType::PoweredLift))
     {
-        auto boosterSpeed = get_booster_speed(curRide->type, (brake_speed << 16));
+        auto boosterSpeed = brake_speed << BrakeSpeedShiftAmount;
         if (boosterSpeed > _vehicleVelocityF64E08)
         {
-            acceleration = GetRideTypeDescriptor(curRide->type).OperatingSettings.BoosterAcceleration
-                << 16; //_vehicleVelocityF64E08 * 1.2;
+            acceleration = BoosterAcceleration << 16; //_vehicleVelocityF64E08 * 1.2;
         }
     }
     else if (rideEntry->flags & RIDE_ENTRY_FLAG_RIDER_CONTROLS_SPEED && num_peeps > 0)
     {
         acceleration += CalculateRiderBraking();
     }
 
-    if ((trackType == TrackElemType::Flat && curRide->type == RIDE_TYPE_REVERSE_FREEFALL_COASTER)
-        || (trackType == TrackElemType::PoweredLift))
-    {
-        acceleration = GetRideTypeDescriptor(curRide->type).OperatingSettings.PoweredLiftAcceleration << 16;
```

Comment:
OperatingSettings.PoweredLiftAcceleration is loaded into BoosterAcceleration and brake_speed is set to 255 under these conditions - this block becomes identical to the booster case.